nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -113,6 +113,17 @@ <h1>
                 {% endif %}
             {% endwith %}
         {% endif %}
+        {% if object.is_metadata_associable_model and perms.extras.view_objectmetadata %}
+            {% with object.associated_object_metadatas.count as om_count %}
+                {% if om_count %}
+                    <li role="presentation"{% if request.GET.tab == 'object_metadatas' %} class="active"{% endif %}>
+                        <a href="{{ object.get_absolute_url }}#object_metadatas" onclick="switch_tab(this.href)" aria-controls="object_metadatas" role="tab" data-toggle="tab">
+                            Object Metadatas {% badge om_count %}
```

Comment:
`metadata` is already a plural noun

```suggestion
                            Object Metadata {% badge om_count %}
```